### DISCUSSION TITLE
Why babel compile JSX for facebook ?

### DISCUSSION THREAD

**Discussion by nissenyeh** [babel/babel#13307] [Q&A] [answered]
Hi guys.

I'm curious about the history of babel and JSX.  JSX is usually used with React  (and it's created by facebook)
Why don't facebook invent it's own compiler but rely on babel to compile JSX?
What's relationship between  babel and Facebook ? Does anyone know?

**Comment by fedeci** [comment, marked as answer]
You can find out more in this [blog post](https://it.reactjs.org/blog/2015/06/12/deprecating-jstransform-and-react-tools.html) from the React website.

**Reply by nissenyeh** [reply]
thanks!
